integrations/unity-csharp/Assets/VDK/VDKCollider.cs

```diff
@@ -0,0 +1,311 @@
+    /*
+     * constructs a mesh of world size width x height in metres, each vertex's z dimension is set according to the contents of the z buffer
+     * triangles are declared in a clockwise direction so face normals are back towards the generating camera
+     */
+    private void MakeSheetMesh(double width, double height, int numVertsX, int numVertsY, float[] depthMap = null)
+    {
+        Vector3[] verts;
+        Vector3[] norms = new Vector3[numVertsX * numVertsY];
+        Vector2[] uv = new Vector2[numVertsX * numVertsY];
+        List<int> tris = new List<int>(numVertsX * numVertsY * 3);
+
+
+
+        //initialise a mesh plane centred at origin 
+        verts = new Vector3[numVertsX * numVertsY];
+        double xSpacing = width / numVertsX;
+        double ySpacing = height / numVertsY;
+        int triCount = 0;
+        for (int i = 0; i < numVertsY; i++)
+        {
+            for (int j = 0; j < numVertsX; j++)
+            {
+                float zPrime = depthMap == null ? 0f : depthMap[i * numVertsX + j];
+                float depth = zBufferToDepth(zPrime);
+                //norms[i * numVertsX + j] = new Vector3(0, 0, -1);
+                uv[i * numVertsX + j] = new Vector2(j / (float)numVertsX, i / (float)numVertsY);
+                verts[numVertsX * i + j] = new Vector3((float)(j * xSpacing - width / 2), (float)(i * ySpacing - height / 2), -depth);
+                if (((i + j) % 2 == 0))
+                {//make triangles at every interior even vert
+                    if (i > 0 && j > 0)
+                    {
+                        tris.Add(numVertsX * i + j); //current vertex
+                        tris.Add(numVertsX * i + j - 1);//vertex left
+                        tris.Add(numVertsX * (i - 1) + j);//vertex above
+                        triCount++;
+                    }
+                    if (i > 0 && j < numVertsX - 1)
+                    {
+                        tris.Add(numVertsX * i + j); //current vertex
+                        tris.Add(numVertsX * (i - 1) + j);//vertex above
+                        tris.Add(numVertsX * i + j + 1);//vertex Right
+                        triCount++;
+                    }
+                    if (j < (numVertsX - 1) && i < (numVertsY - 1))
+                    {
+                        tris.Add(numVertsX * i + j); //current vertex
+                        tris.Add(numVertsX * i + j + 1);//vertex Right
+                        tris.Add(numVertsX * (i + 1) + j);//vertex below
+                        triCount++;
+                    }
+                    if (i < numVertsY - 1 && j > 0)
+                    {
+                        tris.Add(numVertsX * i + j); //current vertex
+                        tris.Add(numVertsX * (i + 1) + j);//vertex below
+                        tris.Add(numVertsX * i + j - 1);//vertex left
+                        triCount++;
+                    }
+                }
+            }
+        }
+        if (laplacianSmoothing)
```

Comment:
This should have an empty line before it, and the braces for this if statement could be removed. The next if statement doesn't have braces, this code should be consistent.